src/test/java/com/google/javascript/gents/singleTests/fields.js

```diff
@@ -11,6 +11,11 @@ var A = function(a) {
   let y = 1;
   /** @type {number} */
   this.z = y + 1;
+
+  this.w;
+  this.w.bar = 'bar';
+
+  this.n = 13;
```

Comment:
:cry: You are right, we need to de-duplicate anyway. I'd like to do that in a separate PR, though.